feedback.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
```

Comment:
hard to read code, I think it can be split into smaller logical parts or functions